examples/event-counters/Examples.EventCounters/Program.cs

```diff
@@ -0,0 +1,41 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics.Tracing;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+// Create EventSources and EventCounters
+EventSource eventSource = new("MyEventSource");
+EventCounter eventCounter = new("MyEventCounter", eventSource);
+PollingCounter pollingCounter = new("MyPollingCounter", eventSource, () => new Random().NextDouble());
```

Comment:
As I remember, all this classes implements `IDisposable`.

Reply:
I added `using` statements to ensure the correct use of these `IDisposable` objects.